tests/python/test_pow.py

```diff
@@ -0,0 +1,22 @@
+import taichi as ti
+
+
+@ti.all_archs
+def test_pow_scan():
+  z = ti.var(ti.f32, shape=())
```

Comment:
Here we can probably test the operator with precisions `ti.f64`, `ti.i32`, `ti.i64` as well.